javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -81,10 +87,7 @@ public static void installBytebuddyAgent(Instrumentation inst) {
    * @return the agent's class transformer
    */
   public static ResettableClassFileTransformer installBytebuddyAgent(
```

Comment:
The PR does not change this API, however could we keep it to make sure the existing tests in vendor distributions work?

Reply:
hey @pavolloffay, can you add some tests to the example distro? that will help clear up what impact this change has on them

Reply:
to do this I will have to copy some classes from the main project. Here is an example of what I use in the Hypertrace agent https://github.com/hypertrace/javaagent/blob/main/testing-common/src/main/java/org/hypertrace/agent/testing/AbstractInstrumenterTest.java#L83. We are not publishing the testing artifact https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1644